css-canvas-medical/product.css

```diff
@@ -0,0 +1,327 @@
+.column-header-main {
+  border-bottom: 2px solid rgb(240, 240, 240);
+}
+
+.row-header {
+  display: flex;
+  width: 87%;
+  margin: auto;
+  height: 70px;
+}
+
+.column-header {
+  width: 100%;
+  margin: auto;
+  display: flex;
+}
+
+.column-nav {
+  width: 120%;
+  position: relative;
+  color: #b3b3b3;
+  font-size: 0.8rem;
+  font-weight: bolder;
+  text-align: center;
+}
+
+.column-nav-blog {
+  width: 80%;
+}
+
+.column-logo {
+  width: 450%;
```

Comment:
😱 😱 😱 😱 😱 😱 😱 

You should _**never**_ need a value higher than 100% for a property. I see you have a few other values that are exceeding 100% in your CSS. WHat are you trying to accomplish with these values? Feel free to send me a message on Slack, and I can help you find a better path. Please search through your stylesheet and find all instances of values over 100% and revise.